example/src/main/java/gaffer/example/gettingstarted/generator/DataGenerator6.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Out of scope for this pull request, but data generator code could be rationalised without reducing the readability of the actual example classes.